### PR TITLE
{bio}[iimpi/2019b] MMseqs2 v11-e1a1c

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2403.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/9164125480556daf03bdcfa0c61db317 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/afea0e8e4cfd515ea7299dc968c4dcd6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3111.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/7479b63184e0fc372dfd8f60601b3f20 for a full test report.

File: easybuild/easyconfigs/m/MMseqs2/MMseqs2-11-e1a1c-iimpi-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+easyblock = 'CMakeMake'
```
Comment (lexming) [inline easybuild/easyconfigs/m/MMseqs2/MMseqs2-11-e1a1c-iimpi-2019b.eb]: Please reinstate original attribution for this easyconfig. See `MMseqs2-10-6d92c-gompi-2019b`.

File: easybuild/easyconfigs/m/MMseqs2/MMseqs2-11-e1a1c-iimpi-2019b.eb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+name = 'MMseqs2'
+version = '11-e1a1c'
+
+homepage = 'https://mmseqs.com'
+description = "MMseqs2: ultra fast and sensitive search and clustering suite"
+
+toolchain = {'name': 'iimpi', 'version': '2019b'}
+
+source_urls = ['https://github.com/soedinglab/MMseqs2/archive/']
```
Comment (lexming) [inline easybuild/easyconfigs/m/MMseqs2/MMseqs2-11-e1a1c-iimpi-2019b.eb]: I would suggest to better use `GITHUB_SOURCE` for simplicity
```suggestion
github_account = 'soedinglab'
source_urls = [GITHUB_SOURCE]
```

**Comment by boegel** [comment]
@lexming Suggestions finally dealt with...

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node376.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/caffea3413389400bedaaed2225642fb for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node101.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/5708d0c8cbbad5e5ff99fa25a5be5451 for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @boegel !

**Pull request merged**